sycl/test-e2e/DeviceCodeSplit/grf.cpp

```diff
@@ -14,6 +14,9 @@
 //   compiler option
 
 // REQUIRES: arch-intel_gpu_pvc
+// https://github.com/intel/llvm/issues/14826
```

Comment:
I don't see this test listed in the issue, let's make sure it's up to date

Reply:
Nice catch. I've updated the GH issue.